invoice/src/main/java/org/killbill/billing/invoice/InvoicePluginDispatcher.java

```diff
@@ -94,7 +94,7 @@ public DateTime priorCall(final LocalDate targetDate,
                               final boolean isRescheduled,
                               final CallContext callContext,
                               // The pluginProperties list passed to plugins is mutable by the plugins
-                              @SuppressWarnings("TypeMayBeWeakened") final LinkedList<PluginProperty> properties,
+                              @SuppressWarnings("TypeMayBeWeakened") final Iterable<PluginProperty> properties,
```

Comment:
NACK. There's a reason why we have a `@SuppressWarnings("TypeMayBeWeakened")` here 😄 

See https://github.com/killbill/killbill/pull/1333/files

Reply:
See https://github.com/killbill/killbill/pull/1696

Reply:
PR has been merged and should address the comment.